package.json

```diff
@@ -2,6 +2,8 @@
   "name": "livesite-2021",
   "version": "0.1.0",
   "private": true,
+  "author": "Team HackBeanpot <team@hackbeanpot.com>",
+  "prod": "gatsby serve",
```

Comment:
just want to double check that all these new package imports and ones below are needed, unable to figure out what they all correspond to. cc'ing @snwessel @daniaalnahdi since y'all set up the packages

Reply:
I suggest avoiding gatsby. We used it for the main HBP website to load data from JSON files, but it's a bit more than we need for the livesite given that we're using Airtable to store data - and would change the way we need to compile things.

Is there a specific reason you added it here? (like is there anything that breaks without it?)